roles/kubernetes/node/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+kube_slice_cgroup: "/{{ kube_slice.split('-') | join('.slice/') }}/"
+system_slice_cgroup: "/{{ system_slice.split('-') | join('.slice/') }}/"
```

Comment:
Not exactly.
It would be `/runtime.slice/` and `/system.slice/`.
This is intented to make the translation from systemd slice units to the correspond cgroup tree in `/sys/fs/cgroup/`

so `runtime.slice` -> `/sys/fs/cgroup/runtime.slice/`
`nested-runtime.slice -> `/sys/fs/cgroup/nested.slice/runtime.slice`
But I'm just seeing now that this does not work exactly that way, instead it should be
`nested-runtime.slice` `/sys/fs/cgroup/nested.slice/nested-runtime.slice`

I'll go fix that

Reply:
Ah yes indeed forgot the `/` but I was more wondering about the split/join which doesn't do anything here AFAIK (?)

Reply:
It doesn't do anything on slice immediately under the root slice (like `kube.slice`).

However, if a kubespray user define `kube_slice: orchestrator-kube.slice` the corresponding cgroup will be 
`/sys/fs/cgroup/orchestrator.slice/orchestrator-kube.slice/` (I had mistakenly assumed it was `orchestrator.slice/kube.slice/`, which the current code reflects.)

However, I need to do more research on that. It seems that maybe the *ReservedCgroup setting are interpreted differently depending on the cgroupDriver used (`cgroupfs/systemd`) and in systemd case the translation from slice to cgroup would be done directly by the kubelet. Not completely sure though so I've asked on slack sig-node ( https://kubernetes.slack.com/archives/C0BP8PW9G/p1729771784322639 )